sli/dictutils.h

```diff
@@ -251,6 +251,36 @@ updateValue( DictionaryDatum const& d, Name const n, VT& value )
   value = getValue< FT >( t );
   return true;
 }
+/* Polymorphic version of update_value.
```

Comment:
Adjust this comment to the same (doxygen) style as the other function-describing comments in `dictutils.h`.